### DISCUSSION TITLE
Supabase Auth

### DISCUSSION THREAD

**Discussion by magick93** [medusajs/medusa#5476] [Feature Requests]
Hello

As Medusa builds on Postges and also provides instructions on how to connect to Supabase, it would be good if it was also possible to use the Supabase Auth.

https://supabase.com/docs/guides/auth

Thanks

**Comment by lecramr** [comment]
+1

**Comment by redjor** [comment]
+1

**Comment by n0rig** [comment]
+1

**Comment by alaistair76** [comment]
+1

**Comment by kunutch** [comment]
+1

**Comment by benyto** [comment]
+1 
Any feedback on this?

**Comment by thehouseisonfire** [comment]
+1
